src/main/resources/templates/start.html

```diff
@@ -7,18 +7,24 @@
     <meta charset="UTF-8">
     <link rel="icon" href="https://www.shareicon.net/data/2017/01/18/872963_game_512x512.png">
     <title>Hobby Collection</title>
-    <link th:href="@{/css/style.css}" type="text/css" rel="stylesheet">
+    <link href="/src/main/resources/static/css/style.css" type="text/css" rel="stylesheet">
```

Comment:
I'd keep this as it were.  It was using proper ThymeLeaf syntax.  The change uses a relative path; most of which won't be exposed once this is compiled and built into a deployable file (war file).